src/main/groovy/org/jetbrains/intellij/IntelliJPlugin.groovy

```diff
@@ -111,6 +112,14 @@ class IntelliJPlugin implements Plugin<Project> {
         def ivyFile = createIvyRepo(project, extension)
         def version = extension.version
 
+        project.configurations.create("provided")
```

Comment:
`provided` is wide-spread configuration name, it's better to set some more specific name, e.g. `intellij-plugins`